iommu_registers.adoc

```diff
@@ -234,6 +234,9 @@ must be supported.
 IOMMU implementations must support the Svnapot standard extension for NAPOT
 Translation Contiguity.
 
+The `PAS` field specifies the width, in bits, of the physical address supported
+by the IOMMU.
+
```

Comment:
The physical address space addressable by the IOMMU is 0 to 2^(capabilities.PAS)-1.